gradle/buildScan.gradle

```diff
@@ -15,29 +15,14 @@
  */
 def env = System.getenv()
 def publishToSlack = gradle.startParameter.projectProperties.publishToSlack != null
+def ci = System.getenv("CI")
 
 gradleEnterprise.buildScan {
-  background {
-    def commitId = env.REVISION
-    try {
-      commitId = 'git rev-parse --verify HEAD'.execute().text.trim()
-    } catch (ignore) {
-      // ignore
-    }
-
-    if (commitId) {
-      link "Source", "https://github.com/ratpack/ratpack/tree/" + commitId
-    }
-  }
-}
-
-if (System.getenv("CI")) {
-  gradleEnterprise.buildScan {
-    termsOfServiceUrl = 'https://gradle.com/terms-of-service'
-    termsOfServiceAgree = 'yes'
-
+    server = "https://ge.ratpack.io"
+    captureTaskInputFiles = true
+    uploadInBackground = !ci
     publishAlways()
-    tag "CI"
+    publishIfAuthenticated()
```

Comment:
Please add a link to this PR that points to how to create a publish key in GE that can be used on CI.

Reply:
Done!

Reply:
Looks like @rhart already it.

Reply:
Yes, that should be done.